libp2p/protocols/connectivity/relay/client.nim

```diff
@@ -89,7 +84,7 @@ proc reserve*(cl: RelayClient,
     pb = encode(HopMessage(msgType: HopMessageType.Reserve))
     msg = try:
       await conn.writeLp(pb.buffer)
-      HopMessage.decode(await conn.readLp(RelayClientMsgSize)).get()
+      HopMessage.decode(await conn.readLp(RelayClientMsgSize)).tryGet()
```

Comment:
Maybe we can use `valueOr` and raise a more specific exception here.

Reply:
It's the only tryGet in this block, so it would be easy to know that the exception is coming from here